### PR TITLE
Fix missed import

### PR DESCRIPTION
**Pull request by lowellrex**
Resolves an issue some Intake users are seeing when they try to load Intake, where [Intake displays a blank page](https://sentry.prod.appeals.va.gov/department-of-veterans-affairs/caseflow/issues/20348/).

It appears as if `intake.js` relied lodash as imported by some other file that was removed in a [PR deployed yesterday afternoon](https://github.com/department-of-veterans-affairs/caseflow/pull/16742/).

I still don't understand why Angel and I did not encounter this issue when logged in as ourselves, but when shadowing affected users we were able to reproduce this issue.

Determined which files were affected by running the following commands to find instances where we used lodash but did not directly import it:
```bash
$> git grep -l "\_\." client/ > uses.txt

$> git grep -l "import _" > imports.txt

$> diff uses.txt imports.txt

18d17
< client/app/components/Table.stories.js
49,55d47
< client/app/images/help/caseflow-cancelmodal.jpg
< client/app/images/help/caseflow-mismatched.jpg
< client/app/images/help/caseflow-mismatched_v2.jpg
< client/app/images/help/dispatch_reference.pdf
< client/app/images/help/vacols-cleardate.png
< client/app/images/help/vbms-addlabel.jpg
< client/app/images/help/vbms-addlabel.png
57c49
< client/app/intake/actions/intake.js
---
> client/app/intake/actions/rampElection.js
85d76
< client/app/queue/CaseList/CaseListActions.js
147,149d137
< client/asset-compile-diagram.png
< client/flow-typed/npm/lodash_v4.x.x.js
< client/karma.conf.js
```

### CODE REVIEW

File: client/app/intake/actions/intake.js [hunk 1]
```diff
@@ -198,7 +199,7 @@ export const submitIntakeReviewRequest = (intakeId, data) => (dispatch) => {
 export const analyticsCallback = (triggerEvent, category, actionName) => {
   triggerEvent(category, actionName, 'any-error');
 
-  _.forEach(
+  forEach(
```
Comment (lowellrex) [inline client/app/intake/actions/intake.js:202]: This is the culprit responsible for the intake page not loading.

File: client/app/queue/CaseList/CaseListActions.js [hunk 2]
```diff
@@ -120,7 +120,7 @@ export const fetchAppealsBySearch = (searchTerm) => (dispatch) => {
       return uniqueVetIdsFromCases(returnedObject);
     }).
     catch((error) => {
-      const backendError = _.get(error.response, 'body');
+      const backendError = get(error.response, 'body');
```
Comment (lowellrex) [inline client/app/queue/CaseList/CaseListActions.js:123]: I noticed this instance of lodash being used without being directly imported while making this change so I figured I would change it while in here. Does not appear to cause the intake issue inspiring this PR.

**Review by yoomlam — APPROVED** [review]
LGTM

**Review by charleyf — APPROVED** [review]
Lowell went through this for the folks on the SWAT call. This looks good to me.

**Comment by yoomlam** [comment]
Should this line be removed? https://github.com/department-of-veterans-affairs/caseflow/blob/fc3c85c8430db89c12e02d399f04a97eea8877aa/client/app/intake/actions/rampElection.js#L4

I doesn't appear to be used.

**Pull request merged**